setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup, find_packages
+
+
+setup(
+    name="xplaneconnect",
+    version="1.0.0",
+    python_requires=">=3",
+    packages=find_packages("Python3/src"),
+    package_dir={"": "Python3/src"},
+    description="XPlaneConnect (XPC) facilitates communication to and from the XPCPlugin.",
```

Comment:
```suggestion
    description="XPlaneConnect (XPC) facilitates communication to and from the XPCPlugin. Note: To use the XPC package, you must first install the XPCPlugin. See https://github.com/nasa/XPlaneConnect/wiki/Getting-Started for instructions on installing the XPCPlugin",
```
I'm afraid that users will install this from pip without understanding that they must first install the XPCPlugin. Hopefully this note will help prevent that.